examples/karpenter/README.md

```diff
@@ -22,10 +22,40 @@ aws eks --region eu-west-1 update-kubeconfig --name ex-karpenter
 kubectl scale deployment inflate --replicas 5
 
 # You can watch Karpenter's controller logs with
-kubectl logs -f -n karpenter -l app.kubernetes.io/name=karpenter -c controller
+kubectl logs -f -n kube-system -l app.kubernetes.io/name=karpenter -c controller
```

Comment:
I can't comment on the line since its not changed - but we should update the README above to remove the Fargate references

Reply:
Check if it's OK. I forgot to remove the Fargate outputs, so docs were not update. Other than that I just rephrased the first paragraph.